shell/platform/windows/flutter_window_winuwp.cc

```diff
@@ -11,10 +11,11 @@ namespace flutter {
 // Multipler used to map controller velocity to an appropriate scroll input.
 static constexpr double kControllerScrollMultiplier = 3;
 
-// TODO(clarkezone): Determine pointer ID in
-// OnPointerPressed/OnPointerReleased/OnPointerMoved in order to support multi
-// touch. See https://github.com/flutter/flutter/issues/70201
-static constexpr int32_t kDefaultPointerDeviceId = 0;
+// Minimum pointer ID that gets emitted by the pointer ID generator.
+static constexpr auto kMinPointerId = 0;
+
+// Maximum pointer ID that gets emitted by the pointer ID generator.
+static constexpr auto kMaxPointerId = 128;
```

Comment:
Use the explicit type for top-level constant declarations -- in this case, `uint32_t`, for pointer IDs. 

We're not particularly consistent about this ourselves, but for the most part, try to restrict use of `auto` to locals where the type is evident from the right hand side..